src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,285 @@
+                    <DesignReviewMemberPill
+                      user={member}
+                      handleClick={() => {
+                        handleRemoveRequiredMember(member);
+                      }}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item sx={{ display: 'flex', alignItems: 'start', marginTop: '7px' }}>
+                <Typography>Optional: </Typography>
+              </Grid>
+              <Grid item>
+                <Grid container>
+                  {optionalMembers.map((member, index) => (
+                    <Grid item key={index}>
+                      <DesignReviewMemberPill
+                        user={member}
+                        handleClick={() => {
+                          handleRemoveOptionalMember(member);
+                        }}
+                      />
+                    </Grid>
+                  ))}
+                </Grid>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+    </Box>
+  );
+
+  const SummaryDetails: React.FC = () => (
+    <Grid container direction="row" alignItems="center" justifyContent="center" columnSpacing={1}>
+      <Grid item xs={3}>
+        <DesignReviewPill
+          icon={<AccessTimeIcon />}
+          isLink={false}
+          displayText={`${meetingDatePipe(designReview.dateCreated)} ${availabilityStartTimePipe(
+            designReview.meetingTimes
+          )}`}
+        />
+      </Grid>
+      <Grid item xs={3}>
+        {editing ? (
+          <TextField
+            variant="standard"
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setLocationState(event.target.value);
+            }}
+            value={locationState}
+            label="Enter Location"
+          ></TextField>
+        ) : (
+          <DesignReviewPill icon={<LocationOnIcon />} isLink={false} displayText={locationState ?? 'Online'} />
+        )}
+      </Grid>
+      <Grid item xs={3}>
+        {editing ? (
+          <TextField
+            variant="standard"
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setDocumentState(event.target.value);
+            }}
+            value={documentState}
+            label="Enter Docs URL"
+          ></TextField>
+        ) : (
+          <DesignReviewPill
+            isLink
+            icon={<DescriptionIcon />}
+            linkURL={documentState ?? ''}
+            displayText={documentState ? 'Questions Doc' : 'No Doc'}
+          />
+        )}
+      </Grid>
+      <Grid item xs={3}>
+        {editing ? (
+          <TextField
+            variant="standard"
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setZoomState(event.target.value);
+            }}
+            value={zoomState}
```

Comment:
the problem is likely because you're trying to update the zoomState literally as you're typing it. How this is going to actually function is that these values will be saved, and only when someone clicks save, an updated payload will be sent to the backend, updating the design review data which will then be updated in the frontend

Reply:
so for now, no need to implement the actual functionality just make it go into and out of edit mode when you click edit/ save

Reply:
also the save button should just replace the edit button when you're in edit mode